com.foss.llamas.poker/lib/src/main/java/com/foss/llamas/poker/domain/StandardCard.java

```diff
@@ -6,7 +6,7 @@ public class StandardCard implements Card {
 	private Suit suit;
 	
 	private boolean wild;
-	private StandardCard(Color color, Rank rank, Suit suit) {
+	StandardCard(Color color, Rank rank, Suit suit) {
```

Comment:
```suggestion
	private StandardCard(Color color, Rank rank, Suit suit) {
```

Instead of changing the constructor, let's use the static factory method `StandardCard.build(Rank, Suit)` because it returns a `Card` object, which is the interface.